tests/test_reverse.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+
+import numpy as np
+
+from audiomentations import Reverse
+
+
+class TestReverse(unittest.TestCase):
+    def test_single_channel(self):
+        samples = np.array([0.5, 0.6, -0.2, 0.0], dtype=np.float32)
+        sample_rate = 16000
+        augmenter = Reverse(p=1.0)
+        samples = augmenter(samples=samples, sample_rate=sample_rate)
+
+        self.assertEqual(samples.dtype, np.float32)
+        self.assertEqual(len(samples), 4)
+
+    def test_multichannel(self):
+        samples = np.array(
+            [[0.9, 0.5, -0.25, -0.125, 0.0], [0.95, 0.5, -0.25, -0.125, 0.0]],
+            dtype=np.float32,
+        )
+        sample_rate = 16000
+        augmenter = Reverse(p=1.0)
+        samples = augmenter(samples=samples, sample_rate=sample_rate)
+
+        self.assertEqual(samples.dtype, np.float32)
```

Comment:
Please add an assertion that checks that the output is reversed as expected when applied to multichannel audio:

```
        assert_array_almost_equal(
            reversed_samples,
            np.array(
                [[0.0, -0.125, -0.25, -0.5, 0.9], [0.0, -0.125, -0.25, 0.5, 0.95]],
                dtype=np.float32,
            ),
        )
```

You may find that the test fails after you add this assertion. In that case, the implementation needs to be fixed. Maybe slice notation is better than np.flipud?

Reply:
Can you tell me what is the source of `assert_array_almost_equal` function?

Reply:
Sorry, I forgot to mention that. It's `from numpy.testing import assert_array_almost_equal`

Reply:
I've fixed that problem and updated test file, please check.